charts/bindplane/templates/bindplane-nats.yaml

```diff
@@ -0,0 +1,362 @@
+{{- if eq .Values.eventbus.type "nats" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "bindplane.fullname" . }}-nats
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "bindplane.name" . }}
+    app.kubernetes.io/stack: bindplane
+    app.kubernetes.io/component: nats
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 3
+  serviceName: {{ include "bindplane.fullname" . }}-nats-cluster-headless
```

Comment:
This needs to match the name of the headless service, and allows the `BINDPLANE_NATS_SERVER_CLUSTER_ROUTES` variable to point to the pod hostnames (all three of them) instead of the service name.